### DISCUSSION TITLE
Limiting Boundary Condition

### DISCUSSION THREAD

**Discussion by RobMacka** [idaholab/moose#20873] [Q&A General] [answered]
Hi MOOSE users,

I'm wondering what is the most sensible way to implement the following condition at the boundaries?
<img width="196" alt="Screenshot 2022-04-26 at 16 35 12" src="https://user-images.githubusercontent.com/73183527/165338082-6079ef74-0ca3-4240-a265-3b5b1ec31b7a.png">

Best Regards,
Robert

**Comment by GiudGiud** [comment, marked as answer]
Hello

Depends on the equation.
Do you have an analytical form that you could integrate such that phi(x=some_number) = some_value (for Dirichlet, similar for Neumann) is equivalent to this limit condition ?
You could also just make a really big mesh and put that calculation on a computing cluster. 

Guillaume
